code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1194,6 +1194,18 @@
 	color = "#F5F5F5"
 	self_consuming = TRUE
 
+/datum/reagent/medicine/corazone/on_mob_add(mob/M)
+	..()
+	if(isliving(M))
```

Comment:
This check can be removed, the signature of `on_mob_add(mob/M)` is ACTUALLY `on_mob_add(mob/living/M)`, because on_mob_add only runs for living mobs as per https://github.com/tgstation/tgstation/blob/960837c16bece813277b73f4e8e4b97a49b857c5/code/modules/reagents/chemistry/holder.dm#L603

Also if you somehow inject reagents into camera mobs we want to get the runtime